### DISCUSSION TITLE
mobile browser support

### DISCUSSION THREAD

**Discussion by bbigras** [emacs-lsp/lsp-gitpod#17] [Ideas]
I don't have a Bluetooth keyboard yet, but being able to use lsp-gitpod on my phone would be awesome.

Is this a goal?

And is there also key collisions when using mobile browsers?

**Comment by yyoncho** [comment]
The goal is to support as much stuff as possible. IMHO even if gtk broadway does not support mobile, we can still support mobile devices once we add web terminal support (most likely via xtermjs).
